server/services/nomisMigrationService.ts

```diff
@@ -102,6 +102,24 @@ export default class NomisMigrationService {
     })
   }
 
+  async getDLQMessageCount(context: Context): Promise<string> {
+    logger.info(`getting DLQ message count`)
+    const token = await this.hmppsAuthClient.getSystemClientToken(context.username)
+    const health = await NomisMigrationService.restClient(token).get<{
```

Comment:
very minor but, since this doesn't require a token you could have used the token in the `context` to avoid the additional call